pom.xml

```diff
@@ -584,7 +584,7 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>3.3.2</version>
+        <version>3.5</version>
```

Comment:
since we do an update, shouldn't we consider latest stable 3.6? or you need a specific version for reasons?